setup.py

```diff
@@ -23,7 +23,7 @@
         'dsnparse',
         'fiona>=1.7.2,<2.*',
         'folium',
-        'ipyleaflet',
+        'ipyleaflet!=0.8.2',
```

Comment:
only this specific version is problematic? othe higher and lower are fine?

Reply:
Yes: https://github.com/jupyter-widgets/ipyleaflet/issues/185